pkg/maps/lbmap/ipv4.go

```diff
@@ -234,8 +235,15 @@ func (s *Service4Value) GetCount() int        { return int(s.Count) }
 func (s *Service4Value) SetRevNat(id int)     { s.RevNat = uint16(id) }
 func (s *Service4Value) GetRevNat() int       { return int(s.RevNat) }
 func (s *Service4Value) RevNatKey() RevNatKey { return &RevNat4Key{s.RevNat} }
-func (s *Service4Value) SetFlags(flags uint8) { s.Flags = flags }
-func (s *Service4Value) GetFlags() uint8      { return s.Flags }
+func (s *Service4Value) SetFlags(flags uint16) {
+	s.Flags = uint8(flags & 0xff)
+	s.Flags2 = uint8(flags >> 8)
+}
+func (s *Service4Value) GetFlags() uint16 {
+	flags := uint16(s.Flags2)
+	flags |= (flags << 8) | uint16(s.Flags)
+	return flags
```

Comment:
Assuming arbitrary input:
* flags = 0x80
* flags2 = 0x01

flags = 0x1 | (0x1 << 8) | (0x80) => 0x0181

I think this should be something like:
```suggestion
	return (uint16(s.Flags2) << 8) | uint16(s.Flags))
```